MkBuildBot/Program.cs

```diff
@@ -81,8 +81,8 @@ private async Task<Task> SlashCommandHandler(SocketSlashCommand command)
             case "gen-build":
                 var gameVersion = (long) command.Data.Options.First(x => x.Name == "game-version").Value;
                 var excludeInlineParam = (long) command.Data.Options.First(x => x.Name == "exclude-inline-bikes").Value;
+                var isDLCCharactersExcluded = (long)command.Data.Options.FirstOrDefault(x => x.Name == "amount").Value == 1;
```

Comment:
I do get why you did the bool check in this line.

However I would love to seperate these here as well. (Like it was done with excludeInlineParam) this gives us the ability if we actually need the param value for whatever reason.